code/__DEFINES/sound.dm

```diff
@@ -7,8 +7,15 @@
 #define CHANNEL_HEARTBEAT 1019 //sound channel for heartbeats
 #define CHANNEL_AMBIENT_EFFECTS 1018
 #define CHANNEL_AMBIENT_MUSIC 1017
-#define CHANNEL_BUZZ 1017
+#define CHANNEL_BUZZ 1016
 #define CHANNEL_ENGINE_ALERT 1015 //Monkestation Edit -- RBMK Reactor Sounds
+#define CHANNEL_SOUND_EFFECTS 1014
+#define CHANNEL_SOUND_FOOTSTEPS 1013
+#define CHANNEL_WEATHER 1012
+#define CHANNEL_MACHINERY 1011
+#define CHANNEL_INSTRUMENTS 1010
```

Comment:
these proxy channels probably have a better way of existing than current, we can't normally group sounds as sounds need a separate audio channel to play, but adding in a mixer_channel that isn't a standard channel causes issues